lib/octokit/client/repositories.rb

```diff
@@ -177,6 +177,19 @@ def delete_repository(repo, options = {})
       end
       alias :delete_repo :delete_repository
 
+      # Transfer repository
+      #
+      # Requires authenticated repository or organization owner of source and destination.
+      #
+      # @see https://developer.github.com/v3/repos/#transfer-a-repository
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @return [Sawyer::Resource] Repository info for the transferred repository
```

Comment:
We should document the optional parameters as well.

Take a look at https://github.com/octokit/octokit.rb/blob/f72f466d6b183d24b461a7c2cbf1c5cbec22ac3d/lib/octokit/client/authorizations.rb#L44-L49 to see how that's done elsewhere.

Reply:
Thank you. I will add the document.